### ISSUE TITLE
Does/should setDefaultChannel replace the `fcm_fallback_notification_channel`? 

### ISSUE THREAD

**Issue by spacesuitdiver** [dpa99c/cordova-plugin-firebasex#490]
Does/should setDefaultChannel replace the `fcm_fallback_notification_channel`? Is there a decent way to use that as the channel? We really just want a default channel and honestly even `setDefaultChannel` isn't working properly.

**Comment by dpa99c** [comment]
As [outlined here](https://stackoverflow.com/a/50683079/777265), `fcm_fallback_notification_channel` is a channel ID created by the Firebase SDK/Android OS.

During plugin initialization, the plugin [creates a default channel](https://github.com/dpa99c/cordova-plugin-firebasex/blob/master/src/android/FirebasePlugin.java#L180) which [uses the configured default ID](https://github.com/dpa99c/cordova-plugin-firebasex/blob/master/src/android/FirebasePlugin.java#L178) which [is set as](https://github.com/dpa99c/cordova-plugin-firebasex/blob/master/src/android/cordova-plugin-firebase-strings.xml#L3) `fcm_default_channel`.


[`setDefaultChannel()`](https://github.com/dpa99c/cordova-plugin-firebasex/blob/master/src/android/FirebasePlugin.java#L1913) enables you to override this default channel with your own defaults including a different ID specified in the arguments.


TL;DR: either specify your own default channel ID and reference it in your notification message payload or use the plugin default as the [example project does](https://github.com/dpa99c/cordova-plugin-firebasex-test/blob/master/messages/notification.json#L12).
